labelbox/schema/bulk_import_request.py

```diff
@@ -25,6 +23,71 @@
 logger = logging.getLogger(__name__)
 
 
+def __make_file_name(project_id: str, name: str) -> str:
+    return f"{project_id}__{name}.ndjson"
+
+
+# TODO(gszpak): move it to client.py
+def __make_request_data(project_id: str, name: str, content_length: int,
+                        file_name: str) -> dict:
+    query_str = """mutation createBulkImportRequestFromFilePyApi(
+            $projectId: ID!, $name: String!, $file: Upload!, $contentLength: Int!) {
+        createBulkImportRequest(data: {
+            projectId: $projectId,
+            name: $name,
+            filePayload: {
+                file: $file,
+                contentLength: $contentLength
+            }
+        }) {
+            %s
+        }
+    }
+    """ % query.results_query_part(BulkImportRequest)
+    variables = {
+        "projectId": project_id,
+        "name": name,
+        "file": None,
+        "contentLength": content_length
+    }
+    operations = json.dumps({"variables": variables, "query": query_str})
+
+    return {
+        "operations": operations,
+        "map": (None, json.dumps({file_name: ["variables.file"]}))
+    }
+
+
+# TODO(gszpak): move it to client.py
+def __send_create_file_command(
+        client, request_data: dict, file_name: str,
+        file_data: Tuple[str, Union[bytes, BinaryIO], str]) -> dict:
+    response = requests.post(
+        client.endpoint,
+        headers={"authorization": "Bearer %s" % client.api_key},
+        data=request_data,
+        files={file_name: file_data})
+
+    try:
+        response_json = response.json()
+    except ValueError:
+        raise labelbox.exceptions.LabelboxError(
+            "Failed to parse response as JSON: %s" % response.text)
+
+    response_data = response_json.get("data", None)
+    if response_data is None:
+        raise labelbox.exceptions.LabelboxError(
+            "Failed to upload, message: %s" % response_json.get("errors", None))
+
+    if not response_data.get("createBulkImportRequest", None):
+        raise labelbox.exceptions.LabelboxError(
+            "Failed to create BulkImportRequest, message: %s" %
+            response_json.get("errors", None) or
+            response_data.get("error", None))
+
+    return response_data
+
+
```

Comment:
they didn't need to be `classmethod`s as did not utilize the `cls` itself.

in which case it came down to if they should remain `staticmethod`s rather than module level private functions.

I leaned towards the latter as matter of style